### DISCUSSION TITLE
Is TIGER worth it to supplement Nominatim 4.4 to improve results?

### DISCUSSION THREAD

**Discussion by marlin-mixon** [mediagis/nominatim-docker#568] [Q&A] [answered]
I built Nominatim 4.4 server and comparing accuracy of past Google geocoding results. I think I'm close to 60% and working to improve it. One thing that helped a lot is tweaking the format of the address line by removing Apt numbers and including Street, Drive, Avenue etc. where possible. The main question I'm wondering now is is it worth it to include TIGER data? My MediaGIS 4.4 installation build time is about nine hours. To add TIGER data for Texas, my state of interest, doubles that to 18 hours so trying to decide if it is worth it. Thanks! Marlin

**Comment by mtmail** [comment, marked as answer]
It's worth adding. It adds house number data (interpolated) where OpenStreetMap doesn't have full coverage yet. I don't think it will add many missing street names.

**Comment by marlin-mixon** [comment]
Thanks, will upgrade it for TIGER.  Marlin
